chain/consensus/hierarchical/actors/subnet/validator.go

```diff
@@ -0,0 +1,65 @@
+package subnet
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/xerrors"
+
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+// Validator encapsulated string to be compatible with CBOR implementation.
+type Validator struct {
+	ID      string
+	Address string
```

Comment:
I would expect this to be a multiaddr almost always, but I am fine with this general approach.

Reply:
fixed